{
  "api": "indices.get",
  "file": "/test/free/data_stream/10_basic.yml",
  "name": "Delete data stream with failure store",
  "origin": "yaml",
  "request": {
    "args": {
      "index": ".fs-failure-data-stream1-2025.10.31-000002"
    }
  },
  "response": {
    "headers": {
      "content-length": "581",
      "content-type": "application/json",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "error": {
        "index": ".fs-failure-data-stream1-2025.10.31-000002",
        "index_uuid": "_na_",
        "reason": "no such index [.fs-failure-data-stream1-2025.10.31-000002]",
        "resource.id": ".fs-failure-data-stream1-2025.10.31-000002",
        "resource.type": "index_or_alias",
        "root_cause": [
          {
            "index": ".fs-failure-data-stream1-2025.10.31-000002",
            "index_uuid": "_na_",
            "reason": "no such index [.fs-failure-data-stream1-2025.10.31-000002]",
            "resource.id": ".fs-failure-data-stream1-2025.10.31-000002",
            "resource.type": "index_or_alias",
            "type": "index_not_found_exception"
          }
        ],
        "type": "index_not_found_exception"
      },
      "status": 404
    },
    "statusCode": 404
  }
}